src/project-builder.ts

```diff
@@ -2,55 +2,44 @@ import { spawnSync } from "node:child_process";
 import { writeFileSync } from "node:fs";
 import { chalk } from "./chalk.js";
 import prompt from "enquirer";
-import sentencer from "sentencer";
 import exampleConfig from "./tableland.config.example.js";
 
 const docsLink = "https://docs.tableland.xyz";
 const githubLink = "https://github.com/tablelandnetwork";
 
 export const projectBuilder = async function () {
-  const choices = [
-    "No, I don't want to create a project right now",
-    "Yes, and I want to control everything each step of the way",
-    "Yes, but setup everything for me and stop asking me questions",
-  ];
+  const choices = ["No", "Yes"];
   // @ts-ignore https://github.com/enquirer/enquirer/issues/379
   const select = new prompt.Select({
     name: "wtd",
-    message:
-      "Couldn't find an existing Tableland project, do you want to create a new project?",
+    message: "Welcome to tableland, do you want to create a new project?",
     choices: [...choices],
   });
 
   const shouldCreate = await select.run();
   if (shouldCreate === choices[0]) {
     console.log(
-      `${chalk.yellow.bold(sentencer.make("OK, have a wonderful {{ noun }}!"))}
+      `${chalk.yellow.bold(
+        "OK, if you change your mind run this again anytime!"
+      )}
   Don't forget to checkout our docs at ${chalk.cyan(docsLink)}
   and star us on github at ${chalk.cyan(githubLink)}`
     );
+
     return;
   }
 
-  let mkdirArtifacts = true;
-  if (shouldCreate === choices[1]) {
-    // @ts-ignore https://github.com/enquirer/enquirer/issues/379
-    const confirmer = new prompt.Confirm({
-      name: "mkdirartifacts",
-      message:
-        "Is it ok to create a directory for Tableland's Validator repository?",
-    });
+  // @ts-ignore https://github.com/enquirer/enquirer/issues/379
+  const confirmer = new prompt.Confirm({
+    name: "mkdirartifacts",
+    message: "Do you want to clone Tableland's Validator repository?",
```

Comment:
is this so that you can build the binary for their system? or to run docker images?

Reply:
This is to enable running the validator via docker.

Reply:
maybe this should be a flag, since I think only core team members would want to do that (ie, to use unreleased go-tableland features)

Reply:
Good call